packages/runtime-core/__tests__/components/KeepAlive.spec.ts

```diff
@@ -740,6 +740,11 @@ describe('KeepAlive', () => {
     const spyUnmounted = vi.fn()
 
     const RouterView = defineComponent({
+      slots: null as unknown as {
+        default: {
+          Component: ComponentPublicInstance
+        }
+      },
```

Comment:
strict behaviour by default, this behaviour is a breaking change, I can make it less strict if needed